lib/tasks.js

```diff
@@ -41,8 +41,11 @@ var removeTimer = function(timeoutId, callback, Zone){
 
 var getGlobalEventHandlersNames = function getGlobalEventHandlersNames() {
 	var names = [];
+  var isWorkerScope = typeof WorkerGlobalScope !== "undefined" &&
```

Comment:
does can-globals have something for workerscope?

https://github.com/canjs/can-globals/blob/master/is-web-worker/is-web-worker.js#L30

Do we want to use this?

Reply:
can't because can-zone/register is used as a super dependency.